subprojects/launcher/src/main/java/org/gradle/launcher/cli/CommandLineActionFactory.java

```diff
@@ -276,7 +277,11 @@ public void run() {
             sb.append(currentVersion.getBuildTime());
             sb.append("%nRevision:     ");
             sb.append(currentVersion.getRevision());
-            sb.append("%n%nGroovy:       ");
+            sb.append("%n%nKotlin DSL:   ");
+            sb.append(currentKotlinDslVersion.getProviderVersion());
```

Comment:
Will the Kotlin DSL version continue to matter after 1.0? Will it continue to be maintained and released separately?

If this matters, then we should keep this, but my understanding is that this may not be necessary soon-ish (you would know)

Reply:
I’d say yes to both, at least for a while.